lms/djangoapps/teams/views.py

```diff
@@ -315,7 +320,7 @@ def post(self, request):
             return Response(CourseTeamSerializer(team).data)
 
 
-class TeamsDetailView(RetrievePatchAPIView):
+class TeamsDetailView(ExpandableFieldViewMixin, RetrievePatchAPIView):
```

Comment:
I love that all you need to do is add the mixin and then all the expand behavior just kicks in!